doc/document_format/document_format_v1.md

```diff
@@ -0,0 +1,96 @@
+
+If a transcribee v1 document contains paragraphs, they MUST be stored in the `children` property of the transcribee v1 document as an array.
+Each array element MUST have the following properties:
+
+- `lang`: The `lang` MUST be a string or null. If not null, it MUST be an ietf language code. It MAY NOT be another value.
+- `speaker`: The id of the speaker of the current paragrpah. It MUST be an integer of null.
+- `alternative_speakers`: A list of other possible speakers for this paragraph. The `alternative_speakers` MUST be an array of integers. It MAY be empty. It MUST be sorted in ascending order.
+- `children`: The `children` MUST be an array of _text atoms_.
+- `type`: MUST be set to the string `paragraph`
+
+If the `speaker` is generated automatically, it SHOULD only be set if there is only one possible speaker for this paragraph.
+If more than one speaker is possible, `speaker` SHOULD be set to `null` and all possible speakers SHOULD be stored in `alternative_speakers`.
+
+### Text Atoms
+
+A _text atom_ describes the smallest unit of text in a transcribee v1 document.
+For example, a few characters of a word, a word of a group of words.
+A _text atom_ MUST be an object with the following properties:
+
+- `text`: The text associated with the snippet. It MUST be a string. It MAY NOT be the empty string. It MAY NOT contain a line terminator as [defined in ECMAScript](https://tc39.es/ecma262/#table-line-terminator-code-points). It MAY otherwise contain the entire range of unicode characters. Applications MAY choose to ignore or not render certain white space characters.
+- `conf`: The confidence that the text matches the snippet. It MUST be a value in the inclusive range of 0 and 1, where 0 marks the lowest possible confidence and 1 marks the highest possible confidence. A text that was corrected by the user SHOULD have a confidence of 1.
+- `conf_ts`: The confidence that the `start`/`end` properties are correct. It MUST be a value in the inclusive range of 0 and 1, where 0 marks the lowest possible confidence and 1 marks the highest possible confidence.
+- `start`: The start of the `text` in the associated audio in seconds. It MUST be `null` or a floating point number. It MUST be `null` if and only if the `end` property is also `null`. If not `null`, it MUST be greater to or equal 0. If not `null`, it MUST be smaller to or equal to the `end` property.
+- `end`: The end of the `text` in the associated audio in seconds. It MUST be `null` or a floating point number. It MUST be `null` if and only if the `start` property is `null`. If not `null`, it MUST be greater to or equal to the `start` property. If not `null`, it MUST be smaller than or equal to the total length of the associated audio.
+
+## Speaker Names
+
+A transcribee v1 document MAY contain a mapping of speaker ids to strings in the `speaker_names` property.
+If this property exists, it MUST be an object.
+
+Each key of this object MUST be an integer representing a speaker id.
```

Comment:
Do we want to use integers as ids? String could be handy for UUIDs

Reply:
I don't have a preference. Integers make it easy to generate a speaker name ("Speaker [ID]") if none is set.

Reply:
I think integers are fine for know (as this is what we are currently doing). If we want to change it in the future, let's do it in a future version.